src/electron/platform/android/rule-information-provider.ts

```diff
@@ -82,6 +82,139 @@ export class RuleInformationProvider {
+                'ClassNameCheck',
+                `${this.ruleLinkBaseUrl}/unsupported-item-type/`,
+                'View objects must have valid UI class names.',
+                [link.WCAG_4_1_2],
+                () => ({
+                    howToFixSummary:
+                        "Provide a className value within the element's AccessibilityNodeInfo that: 1. Closely describes the element's function, 2. Matches a class that extends android.view.View, 3. Is provided with the Android SDK or support libraries, and 4. Is as far down Android's class hierarchy as possible.",
+                    richResolution: {
+                        labelType: 'check',
+                        contentId: 'android/atfa/ClassNameCheck',
+                    },
+                }),
+                this.getStandardResultStatus,
+            ),
+            ClickableSpanCheck: new RuleInformation(
+                'ClickableSpanCheck',
+                `${this.ruleLinkBaseUrl}/link-purpose-unclear/`,
+                "A TextView widget must not contain ClickableSpan objects that aren't URLSpans with absolute URLs.",
+                [link.WCAG_4_1_2],
+                () => ({
+                    howToFixSummary:
+                        'Implement the link using URLSpan or Linkify. If you use URLSpan, provide a non-null absolute URL (such as https://example.com/page.html), not a relative URL (such as /page.html).',
+                    richResolution: {
+                        labelType: 'check',
+                        contentId: 'android/atfa/ClickableSpanCheck',
+                    },
+                }),
+                this.getStandardResultStatus,
+            ),
+            DuplicateClickableBoundsCheck: new RuleInformation(
+                'DuplicateClickableBoundsCheck',
+                `${this.ruleLinkBaseUrl}/duplicate-clickable-views/`,
+                'Clickable View objects must not have identical boundaries.',
+                [link.WCAG_4_1_2],
+                () => ({
+                    howToFixSummary:
+                        'When clickable Views are nested, implement click handling so that only one View handles clicks for any single action. If a View that is clickable by default (such as a button) is not intended to be clickable, remove its OnClickListener, or set android:clickable="false".',
+                    richResolution: {
+                        labelType: 'check',
+                        contentId: 'android/atfa/DuplicateClickableBoundsCheck',
+                    },
+                }),
+                this.getStandardResultStatus, // TODO: Confirm
+            ),
+            DuplicateSpeakableTextCheck: new RuleInformation(
+                'DuplicateSpeakableTextCheck',
+                `${this.ruleLinkBaseUrl}/duplicate-descriptions/`,
+                'Clickable View objects that do not perform the same function must not have the same speakable text.',
+                [link.WCAG_4_1_2],
```

Comment:
Should this be 4.1.1?

Reply:
Good question. I discussed with @iamrafan, and we both feel that 4.1.2 is more appropriate because the error is about context (4.1.2), not about parsing (4.1.1). That said, @ferBonnin's docs PR uses 4.1.1, so we need to reconcile the differences

Reply:
agree it should be 4.1.2, will change the docs side

Reply:
Docs have been updated to 4.1.2, so the code is now in sync. Leaving as is.